omero/developers/json-api.rst

```diff
@@ -1278,6 +1278,169 @@ you can also show ROIs on an Image using this URL:
     GET     /api/v0/m/images/{image_id}/rois/
 
 
+Experimenters and ExperimenterGroups
```

Comment:
in the UI we talk about Experimenter and Groups
so maybe indicate that when retrieving via the API we have to use the "model" object ExperimenterGroup.